gui/app/(tabs)/registration.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState } from 'react';
+import { View, Text, TextInput, StyleSheet, TouchableOpacity, Image } from 'react-native';
+import * as ImagePicker from 'expo-image-picker';
+
+interface RegistrationFormData {
+    firstName: string;
+    lastName: string;
+    username: string;
+    email: string;
+    password: string;
+    profilePicture: string | null;
+}
+
+const RegistrationScreen: React.FC = () => {
+    const [formData, setFormData] = useState<RegistrationFormData>({
+        firstName: '',
+        lastName: '',
+        username: '',
+        email: '',
+        password: '',
+        profilePicture: null,
+    });
```

Comment:
This needs to be updated to match up with the /user/create endpoint. It expects: 
* Name
* Email Address
* Password

Keep the profile picture, we'll update the API for that. 

I don't think we need to have a "username" different from name. People should be using their real names, not some free-text whatever they want sort of thing.

We can talk about it more tomorrow during scrum.